starlark/eval_test.go

```diff
@@ -890,3 +891,25 @@ func TestExecutionSteps(t *testing.T) {
 		t.Errorf("execution returned error %q, want cancellation", err)
 	}
 }
+
+// TestDeps fails if the interpreter proper (not the REPL, etc) sprouts new external dependencies.
+// We may expand the list of permitted dependencies, but should do so deliberately, not casually.
+func TestDeps(t *testing.T) {
+	cmd := exec.Command("go", "list", "-deps", ".")
+	out, err := cmd.Output()
+	if err != nil {
+		t.Fatalf("'go list starlark' failed: %s", err)
```

Comment:
Maybe `Skipf` here instead? If the test is cross-compiled and run in an environment without the Go toolchain, it probably shouldn't fail because of this.

Reply:
Done---although now I worry about the check failing silently.